personalize.module

```diff
@@ -345,7 +346,7 @@ function personalize_start_agents() {
   $all_agents = personalize_agent_load_multiple();
   foreach ($all_agents as $agent_name => $agent) {
     $status = personalize_agent_get_status($agent_name);
-    if ($status == PERSONALIZE_STATUS_PAUSED || $status == PERSONALIZE_STATUS_NOT_STARTED) {
+    if ($status == PERSONALIZE_STATUS_SCHEDULED) {
```

Comment:
Hmm, I think this actually opens up a discrepancy - in theory the personalize_agent_start_now() function could return true on the basis of something other than scheduling, i.e. using the agent's startNow() method.